packages/query-core/src/tests/query.test.tsx

```diff
@@ -200,7 +191,6 @@ describe('query', () => {
     expect(queryFn).toHaveBeenCalledTimes(1)
     const args = queryFn.mock.calls[0]![0]
     expect(args).toBeDefined()
-    // @ts-expect-error page param should be undefined
     expect(args.pageParam).toBeUndefined()
```

Comment:
it shouldn't be necessary to remove this 🤔 . With this ts-expect-error, we were testing that `args.pageParam` is not accessible on TS level. Is this now because `vi.fn` is not typed anymore ?

Reply:
🤔 taking a look. The `vi.fn` function does support generics, they're just structured differently than `jest.fn` generics...

Reply:
Oof looks like they're just backwards - let me comb out the places I removed generics quick....